examples/trials/ir_enas/trial_code/config.yml

```diff
@@ -0,0 +1,23 @@
+authorName: Unknown
+experimentName: enas_macro
+trialConcurrency: 1
```

Comment:
Can the number of trialConcurrency be bigger? If not, just place a note here and in the doc.